.github/workflows/go.yml

```diff
@@ -0,0 +1,70 @@
+name: Go Build
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+        aws-region: us-east-2
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2.0.0
+
+    - name: Check out submodules
+      uses: textbook/git-checkout-submodule-action@2.0.0
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+    - name: Build
+      run: go build -v ./...
+    
+    - name: Test with coverage
+      run: go test -v ./... -coverprofile coverage.txt
+      
```

Comment:
Need to run integration test as well:

```
go test -run Integration
```

Reply:
Integration tests are already covered by `go test`